nixos/modules/profiles/macos-builder.nix

```diff
@@ -62,11 +65,11 @@ in
 
       # This installCredentials script is written so that it's as easy as
       # possible for a user to audit before confirming the `sudo`
-      installCredentials = pkgs.writeShellScript "install-credentials" ''
-        KEYS="''${1}"
-        INSTALL=${hostPkgs.coreutils}/bin/install
-        "''${INSTALL}" -g nixbld -m 600 "''${KEYS}/${user}_${keyType}" ${privateKey}
-        "''${INSTALL}" -g nixbld -m 644 "''${KEYS}/${user}_${keyType}.pub" ${publicKey}
+      installCredentials = let
+        install = hostPkgs.coreutils}/bin/install;
+      in pkgs.writeShellScript "install-credentials" ''
+        "${install}" -g nixbld -m 600 "''${1}/${user}_${keyType}" ${privateKey}
+        "${install}" -g nixbld -m 644 "''${1}/${user}_${keyType}.pub" ${publicKey}
```

Comment:
I don't feel like this improves the readability of the Nix code, but at the expense of the readability of the generated script

I think whether or not the `install` should be inlined is debatable, but definitely the change to inline `KEYS` feels like a clear regression for readability